pkg/webhook/certs.go

```diff
@@ -89,24 +89,22 @@ func (cr *certRotator) Start(stop <-chan (struct{})) error {
 	// can be bootstrapped, otherwise manager exits before a cert can be written
 	crLog.Info("starting cert rotator controller")
 	defer crLog.Info("stopping cert rotator controller")
-	if restart, err := cr.refreshCertIfNeeded(); err != nil {
+	if refreshed, err := cr.refreshCertIfNeeded(); err != nil {
 		crLog.Error(err, "could not refresh cert on startup")
 		return errors.Wrap(err, "could not refresh cert on startup")
-	} else if restart {
-		crLog.Info("certs refreshed, restarting server")
-		return nil
+	} else if refreshed {
+		crLog.Info("certs refreshed on startup")
 	}
 	ticker := time.NewTicker(rotationCheckFrequency)
 
 tickerLoop:
```

Comment:
we can drop the `tickerLoop` label

Reply:
Thanks for the review :) 
Doesn't `case <- stop` still need it?

Reply:
oops, you are correct.